### PR TITLE
[SCB-1851] config-kie: support enable property

### PR DESCRIPTION
**Pull request by GuoYL123**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/model/KVDoc.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 public class KVDoc {
 
-  @JsonAlias("_id")
+  @JsonAlias("id")
```
Comment (liubao68) [inline dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/model/KVDoc.java]: Can be deleted

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java [hunk 2]
```diff
@@ -50,6 +50,8 @@
 
   private static final String LABEL_VERSION = "version";
 
+  private static final String STATUS_ENABLE = "enabled";
```
Comment (liubao68) [inline dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java]: STATUS_ENABLED

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/model/KVDoc.java (hunk 1 above)
Comment (GuoYL123) [reply dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/model/KVDoc.java]: done

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java (hunk 2 above)
Comment (GuoYL123) [reply dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java]: done

**Pull request merged**